modules/standard/Map.chpl

```diff
@@ -645,6 +645,18 @@ module Map {
       return true;
     }
 
+    deprecated "'Map.set' is deprecated. Please use 'Map.replace' instead."
+    proc set(k: keyType, in v: valType): bool {
+      _enter(); defer _leave();
```

Comment:
It'd be better to just have this call the new method rather than duplicating the code, especially since we don't expect the behavior to differ between the two (aside from whether it throws, but you can still have this method call `replace` and just halt when an error would be thrown)

Reply:
Makes sense, thank you!